### PR TITLE
Create invitation.yml

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .github/workflows/invitation.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    types: [labeled]
+
+jobs:
+  automate_invite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invite on label
+        uses: vj-abigo/invite-on-label@v1.2
+        with:
+          organization: PUGroups
```
Comment (shuklaritvik06) [inline .github/workflows/invitation.yml:12]: Org should be SidHacksCommunity

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]

File: .github/workflows/invitation.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+          label: invite me to the organization
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/Programmers-Universal-Group/people<br>'
```
Comment (shuklaritvik06) [inline .github/workflows/invitation.yml:15]: Here also change the link to <https://github.com/orgs/SidHacksCommunity/people>

**Review by ghost — COMMENTED** [review]
Made the suggested changes!
sorry it happened since even I have my own org profile so...

**Review by shuklaritvik06 — APPROVED** [review]
LGTM! :)

**Pull request closed without merging**